tests/breadcrumbs.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@playwright/test';
+
+test('breadcrumbs should render the list of values as pills', async ({ page }) => {
+  await page.goto('/test.html');
+
+  // GIVEN a "crumbs" attribute has been applied to the v-breadcrumbs element
+  // AND the value passed to the "crumbs" attribute is a comma-separated list of strings
+  // WHEN the element is rendered
+  // THEN the breadcrumbs should render the list of values as pills
+
+  const breadcrumbs = page.getByText('Chocolate Chip Oatmeal Raisin')
+  await expect(breadcrumbs).toBeVisible()
+  await expect(breadcrumbs).toHaveCSS('border-color', 'rgb(0, 0, 0)')
```

Comment:
I don't think we should be testing CSS yet (if ever). It's not very valuable for detecting bugs and creates a bit of overheard when we update styles. I'd recommend removing lines 13-14

Reply:
Makes sense!  Thanks, removed.